plugins/domains/functions

```diff
@@ -66,15 +66,13 @@ domains_add() {
   verify_app_name "$1"
   local APP="$1"; local APP_VHOST_PATH="$DOKKU_ROOT/$APP/VHOST"
 
-  if [[ $(egrep -w "^${2}$" "$APP_VHOST_PATH" > /dev/null 2>&1; echo $?) -eq 0 ]]; then
-    dokku_log_fail "$2 is already defined for $APP"
-    exit 1
-  fi
-  shift 1
```

Comment:
@michaelshobbs Ah ok, thanks! Tests looked very unrelated xD (But that might be a bats issue)